### ISSUE TITLE
rubygem-arr-pm-1:0.0.9 conflicts with file /usr/bin

### ISSUE THREAD

**Issue by ryandoherty** [jordansissel/fpm#838]
(Sorry if this should be a ticket on the rubygem arr-pm repo, not sure where to file this)

Running CentOS7 and FPM 1.3.2 I built rubygem-arr-pm via `fpm -s gem -t rpm arr-pm`, which gave me `rubygem-arr-pm-1:0.0.9-1.noarch`. But when I try to install it yum says it conflicts with package filesystem-3.2-18.el7.x86_6:

`Transaction check error:
  file /usr/bin from install of rubygem-arr-pm-1:0.0.9-1.noarch conflicts with file from package filesystem-3.2-18.el7.x86_64`

I'm not sure where the issue is (FPM or arr-pm gem). From what I've found via googling other packages that have this problem were due to them listing /usr/bin in their specfiles. Removing it from the specfile is supposed to fix the problem.

Any advice on how to fix this myself locally would be helpful too, thanks!

**Comment by jordansissel** [comment]
hmm, fpm shouldn't be trying to claim ownership of base directories like that.

Can you try building this with `--rpm-auto-add-directories` ? I *think* that fixes it. Let me know either way :)

**Comment by ryandoherty** [comment]
Yup, that fixes it! Thanks! I read the wiki on what `--rpm-auto-add-directories` means and I don't think I understand how it fixes this problem. Hoping I can learn what this all means so if I run into similar problems I can fix them.
